### ISSUE TITLE
Request to add fileno() to Logger Class

### ISSUE THREAD

**Issue by theelefant** [Comfy-Org/ComfyUI-Manager#117]
I'd like to request the addition of a `fileno()` method to the `Logger` class in `prestartup_script.py`:

```python
def fileno(self):
    try:
        if self.is_stdout:
            return original_stdout.fileno()
        else:
            return original_stderr.fileno()
    except AttributeError:
        # Handle error
        raise ValueError("The object does not have a fileno method")
```

This is to avoid the following error raised by other ComfyUI nodes which require the `fileno()` method:

```javascript
AttributeError: 'Logger' object has no attribute 'fileno'
```

**Comment by ltdrdata** [comment]
This is patched on 7d6109aa6c0f6f1d482cebe314aa2939044a5f39

**Comment by Nuked88** [comment]
Thanks!
